django_meili/models.py

```diff
@@ -170,11 +170,7 @@ def meili_serialize(self):
             )
         )[0]
 
-        if getattr(self.MeiliMeta, "include_pk_in_search", False):
-            data = dict()
-            data[self.MeiliMeta.primary_key] = getattr(self, self.MeiliMeta.primary_key)
-            serialized_model["fields"].update(data)
-
+        serialized_model["fields"][self.MeiliMeta.primary_key] = getattr(self, self.MeiliMeta.primary_key)
```

Comment:
I'm not necessarily saying get rid of the check, but just condense the body of the code into the one statement. Sorry for the confusion.

Reply:
no worries im a little busy atm so doing a few things at once.

Reply:
✅